src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -139,7 +140,7 @@ public sealed class RegisterPSSessionConfigurationCommand : PSSessionConfigurati
         ## Replace the SDDL with any groups or restrictions defined in the PSSessionConfigurationFile
         if($? -and $configTableSddl -and (-not $isSddlSpecified))
         {{
-            $null = Set-PSSessionConfiguration -Name $pluginName -SecurityDescriptorSddl $configTableSddl -Force:$force
+            $null = Set-PSSessionConfiguration -Name $pluginName -SecurityDescriptorSddl $configTableSddl -NoServiceRestart:$NoRestart -Force:$force
```

Comment:
Makes sense to write a verbose message if it is Windows PowerShell session?
"Windows PowerShell ignores the NoRestart switch."

Reply:
Created doc issue for 5.1 to 7.4 https://github.com/MicrosoftDocs/PowerShell-Docs/issues/11164